HACKING.md

```diff
@@ -210,11 +210,14 @@ If `junitTags` system property isn't defined, all tests without an explicitly de
 
 | Name | Description |
 | :---: | :---: |
-| acceptance | Marks tests executed on Travis | 
+| travis | Marks tests executed on Travis |
+| acceptance | Acceptance tests |
 | regression | Regression tests |
+| upgrade | Execute upgrade tests |
+| pr | Execute subset of tests for pull requests |
```

Comment:
How does this differ from acceptance?

Reply:
Going to remove `pr` tag, make sense to me, that acceptance should be +/- the same.